docs/installation/production/manual/configuration/advanced/social_auth.rst

```diff
@@ -624,6 +624,22 @@ For more detailed information about using Okta social authentication see the fol
 OneLogin
 --------
 
+.. important::
+
+  Currently the ``OneLogin`` backend requires an older version (4.0.2) of the ``social-auth-core`` library than is currently installed by default with Tethys. To use the ``OneLogin`` backend you will need to ensure that your environment has ``social-auth-core==4.0.2``.
+
+  You can install the compatible version of ``social-auth-core`` into an existing environment with the following command:
+
+  .. code-block:: bash
+
+      conda install --experimental-solver libmamba -c conda-forge social-auth-core=4.0.2 openssl=1
+
+  Or you can ensure that you get the compatible version when you create your environment with this command:
+
+  .. code-block:: bash
+
+      conda create --experimental-solver libmamba -n tethys -c tethysplatform -c conda-forge tethys-platform social-auth-core=4.0.2
+
```

Comment:
@ezrajrice will either of these approaches work for your use case?

Reply:
@ezrajrice Any thoughts? This solution should give us the most flexibility for different environments, and should still allow you to specify the environment that you need while the OneLogin backend is being updated.

Reply:
This is a good solution. I don't like having to do one-off fixes, but this will work until we can sort out #881